configure.ac

```diff
@@ -562,9 +562,14 @@ AC_ARG_ENABLE([debug],
               [enable_debug=no])
 if test "x$enable_debug" != xyes; then
    AM_CPPFLAGS="$AM_CPPFLAGS -DNDEBUG"
+   AC_DEFINE([SYM_PRIVATE], [static], [Control visibility of symbols to be used only inside a compilation unit.])
+   AX_CHECK_COMPILE_FLAG([-Wno-error=missing-prototypes], [AM_CFLAGS="$AM_CFLAGS -Wno-error=missing-prototypes"], , [-Werror])
+   AX_CHECK_COMPILE_FLAG([-Wno-missing-prototypes], [AM_CFLAGS="$AM_CFLAGS -Wno-missing-prototypes"], , [-Werror])
```

Comment:
These compiler flag additions are in the wrong branch, see the CI failure.

Reply:
Thanks. Fixed. I wondered what went wrong, but couldn't see it …